src/app/ngrx/reducers/episode-alltime-downloads.reducer.ts

```diff
@@ -18,36 +26,30 @@ export function reducer(
 
     case ActionTypes.CASTLE_EPISODE_ALLTIME_DOWNLOADS_LOAD: {
       const { guid, podcastId } = action.payload;
-      return {
-        ...adapter.upsertOne({
+      return adapter.upsertOne(
```

Comment:
The ngrx/entity upsert interface changed to no longer use that changes business and just take the whole entity record.